### PR TITLE
Release 3.6.00

### PR DESCRIPTION
**Pull request by ndellingwood**

### CODE REVIEW

**Comment by ndellingwood** [comment]
Hold merge until pass and acceptance of trilinos/Trilinos#10253

**Comment by ndellingwood** [comment]
@e10harvey for PRs for merge to master we should test against the corresponding release PR/branch of Kokkos, is there an easy way to point testing to a different branch than develop for this type of case?

**Comment by e10harvey** [comment]
> @e10harvey for PRs for merge to master we should test against the corresponding release PR/branch of Kokkos, is there an easy way to point testing to a different branch than develop for this type of case?

I opened a PR to your release-3.6.00 branch here: https://github.com/ndellingwood/kokkos-kernels/pull/3 and tested the changes here: https://github.com/e10harvey/kokkos-kernels/pull/10. Please let me know if you need anything else.

**Comment by ndellingwood** [comment]
Thanks @e10harvey ! Do I need to approve/merge any of the PRs?

**Comment by e10harvey** [comment]
> Thanks @e10harvey ! Do I need to approve/merge any of the PRs?

I would suggest merging https://github.com/ndellingwood/kokkos-kernels/pull/3. That being said, I see a build failure in https://github.com/e10harvey/kokkos-kernels/pull/10 due to `bhalf_t` being undefined. I assume that kokkos:master has not been updated with the bhalf changes yet. Would it be better if I disable the GitHub action CI checks on our master branch or should the checks be updated to use the release-3.6.00 source branch from Kokkos Core?

**Comment by ndellingwood** [comment]
> I assume that kokkos:master has not been updated with the bhalf changes yet

I believe so (bhalf is part of 3.6.0 right? in that case, the PR has not merged yet to master with the changes)

> Would it be better if I disable the GitHub action CI checks on our master branch or should the checks be updated to use the release-3.6.00 source branch from Kokkos Core?

It would be nice to temporarily test against release-3.6.00, but if that gets messy to toggle between branches then we can just disable the github action ci check

**Comment by ndellingwood** [comment]
@e10harvey https://github.com/ndellingwood/kokkos-kernels/pull/3 merged, thanks for your help

File: .github/workflows/format.yml
Comment (e10harvey) [inline .github/workflows/format.yml:6]:
```suggestion
```
disable the CI checks on PRs targeting master.

File: .github/workflows/osx.yml
Comment (e10harvey) [inline .github/workflows/osx.yml:6]:
```suggestion
```
disable the CI checks on PRs targeting master.

**Comment by crtrott** [comment]
Retest this please

**Comment by lucbv** [comment]
@ndellingwood is there anything we should do before this gets merged?

**Comment by ndellingwood** [comment]
@lucbv just needs an approval then we're ready to merge :)

The follow-on step to merge will be to tag the release, @crtrott are you planning to do that?

**Review by lucbv — APPROVED** [review]
Thanks everyone for all the work that went in this release!

**Pull request merged**
